types/stylelint/index.d.ts

```diff
@@ -252,6 +254,16 @@ declare namespace stylelint {
 		fixable?: boolean;
 	};
 
+	type FixerArguments = { [x: string]: any };
+
+	export type Fixer = (args?: FixerArguments) => string | void;
```

Comment:
note

`string` is to cover the `text` upcoming exposition
i.e. probably not in this PR but it's planned